packages/docusaurus-theme-classic/src/theme/SearchModal/index.tsx

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useState} from 'react';
+import styles from './style.module.css';
+import {useBaseUrlUtils} from '@docusaurus/useBaseUrl';
+import Link from '@docusaurus/Link';
+import {useSearch} from '@theme/useSearch';
+import Fuse from 'fuse.js';
+import {TrySearchIcon, NoResultIcon} from './icon';
+
+const ALLOWED = 30;
```

Comment:
allowed what? I have no idea what this variable is about at first sight. Looks like a name for a boolean to me